docs/guides/weaviate/reconfigure/overview.md

```
@@ -0,0 +1,46 @@
+title: Weaviate Reconfigure Overview
+menu:
+  docs_{{ .version }}:
+    identifier: weaviate-reconfigure-overview
+    name: Overview
+    parent: weaviate-reconfigure
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)